apps/quickstart-integrations-monday-code/index.js

```diff
@@ -87,6 +88,35 @@ app.post(
   }
 );
 
+
+app.post(
```

Comment:
Sorry fi I'm missing something obvious – how is this supposed to be used? Will an external service publish to these endpoints?

Reply:
@yuhgto it's only a way to show end to end the usage of a queue. In a real scenario we will only have:
```
await produceMessage({ message });
```

from within another controller as part of an async flow. i.e. automation webhook that triggers an integration that has a lot of "work" to be done, such as creating many boards with items in monday.

Reply:
awesome. i think there's probably a better way to show this in an example but i don't think it's worth investing in now. let me think about it :)